app/Traits/HasPointsAttribute.php

```diff
@@ -15,6 +15,11 @@ trait HasPointsAttribute
      */
     public function validatePoints(array $points)
     {
+        // Gaps are represented as empty arrays
+        if (empty($points)) {
+            return;
+        }
+
```

Comment:
This case is only relevant for video annotations so I would rather change `validatePoints()` of the VideoAnnotation model. It should only call `validatePoints()` of the trait with non-empty points arrays but still make sure that the gaps make sense (i.e. there must be filled points arrays before and after the gap).